graphene_django/filter/fields.py

```diff
@@ -59,7 +60,12 @@ def resolve_queryset(
             connection, iterable, info, args
         )
         filter_kwargs = {k: v for k, v in args.items() if k in filtering_args}
-        return filterset_class(data=filter_kwargs, queryset=qs, request=info.context).qs
+        filterset = filterset_class(
+            data=filter_kwargs, queryset=qs, request=info.context
+        )
+        if filterset.form.is_valid():
+            return filterset.qs
+        raise ValidationError(filterset.form.errors.as_json())
```

Comment:
To support Python 2.7 and django-filter 1.1 that `form` property is needed. In Graphene-Django v3 we use django-filter 2.x which allows to call `is_valid` and `errors` directly on filterset instances. We may want to refactor that for v3 as it slightly improves readability.